packages/compiler-cli/ngcc/test/utils_spec.ts

```diff
@@ -50,6 +52,121 @@ describe('FactoryMap', () => {
   });
 });
 
+describe('getTsHelperFnFromDeclaration()', () => {
+  const createFunctionDeclaration = (fnName?: string) => ts.createFunctionDeclaration(
+      undefined, undefined, undefined, fnName, undefined, [], undefined, undefined);
+  const createVariableDeclaration = (varName: string) =>
+      ts.createVariableDeclaration(varName, undefined, undefined);
+
+  it('should recongize the `__assign` helper as function declaration', () => {
```

Comment:
```suggestion
  it('should recognize the `__assign` helper as function declaration', () => {
```

And throughout the file :-)